test/integration/rules/submit-button/submit-button.html

```diff
@@ -0,0 +1,24 @@
+<form id="pass1" action="#" method="post">
+  <input></input>
+  <input type="submit"></input>
+</form>
+
+<form id="pass2" action="#" method="post">
+  <input></input>
+  <img type="submit"></input>
+</form>
+
+<form id="pass3" action="#" method="post">
+  <input></input>
```

Comment:
As highlighted by GitHub, there is no need for a closing tag for `input`

Reply:
Edited the HTML code to remove the closing `input` tag.